common/presenters/allocations-to-table-component.js

```diff
@@ -27,11 +27,15 @@ function _byAdded(totalSlots, unfilledSlots) {
 function allocationsToTableComponent({
   showFromLocation = false,
   showRemaining = false,
+  query,
+  enableSorting = true,
 } = {}) {
   const tableConfig = [
     {
       head: {
-        text: 'collections::labels.move_size',
+        sortable: enableSorting,
```

Comment:
I find the `is` prefix pattern quite nice for things like this. Makes it easy to see what properties are a boolean. You could then actually use that same name for the argument as it fits both scenarios and pass it directly in here. And I think it might help to give a clearer name to `key` to make it more obvious that it is referring to the sort key:

```javascript
function allocationsToTableComponent({
  ...
  isSortable = true,
})

...

head: {
  isSortable,
  sortKey: 'moves_count',
  ...
}
```